### DISCUSSION TITLE
Sample script to copy media files

### DISCUSSION THREAD

**Discussion by jiangweiatgithub** [OrchardCMS/OrchardCore#16121] [Q&A]
Hi,

I was just wondering about how to enter the Javascript code into a Script task, for example, copying disk files frome one folder to another. I tried the following script, but nothing happened; it does doe what I want it to when running the first 5 lines in a standalone js script file.
```javascript
var myObject;
myObject = new ActiveXObject("Scripting.FileSystemObject");
var files = "E:\\repos\\javascript\\*.txt";
var targetfolder = "E:\\repos\\javascript\\sub\\";
myObject.CopyFile(files, targetfolder,1);
setOutcome('Done');
```
And how can I pass into a script like the above  a string returned by a Liquid expression, such as the following?
```liquid
{{ Workflow.Input.ContentItem.Content.UserRegister.MyPhoto.Paths }}
```
TIA!
Wei

**Comment by sebastienros** [comment]
You can't invoke custom objects from script tasks. They are supposed to be safe. What you need is a custom method that would be provided to the script engine, to move media. This could be done as part of a PR in the Media module, search for Global Method Provider examples.
